okhttp-tests/src/test/java/okhttp3/EventListenerTest.java

```diff
@@ -43,18 +47,27 @@
 import static org.junit.Assert.fail;
 
 public final class EventListenerTest {
-  @Rule public final MockWebServer server = new MockWebServer();
```

Comment:
I've made a small adjustment to `SocksProxy` so that this can continue to be a `@Rule`.